src/models/thelio-mega-r4-n3/external-overview.md

```diff
@@ -0,0 +1,35 @@
+# Thelio Mega (External Overview)
+
+## Back overview:
+
+![Back View](./img/ports-back.webp)
```

Comment:
The DisplayPort is an input for passthrough to USB-C. It might be good to clarify that, in order to reduce misunderstandings. Something like "DisplayPort Input" might be a good label.

Reply:
I should've noticed that that wasn't finished. Corrected the label & added info about it on the page.